Pa3/Durchführung/Project/src/app/State/SetParameters.c

```diff
@@ -16,51 +16,40 @@
 #include "app/EventEnum.h"
 
 /* CONSTANTS **************************************************************************************/
-
+#define OFFSET_FOR_CHAR 48
 /* MACROS *****************************************************************************************/
 
 /* TYPES ******************************************************************************************/
 
 /* PROTOTYPES *************************************************************************************/
-/** Switch to the next paramSet. If the last paramSet was active, activate the first paramSet
- * @return nothing
-*/
-static void nextParamSet(void);
-
-/** Display the index of the current paramSet on the oled display
- * @return nothing
-*/
-static void displayParamSetIdx(void);
 
 /* VARIABLES **************************************************************************************/
-static UInt8 gParamSetIdx;           /** internal parametersetindex*/
-static ParamSet gParamSetArray[3];    /** array of parameterset, access via gParamSetIdx*/
+static UInt8 gParamSetIdx = 0;           /**< internal parametersetindex*/
+static ParamSet gParamSetArray[3] = {{255, 5, 5, 5 ,5}, 
+{150, 3, 3, 3, 3}, {75, 1, 1, 1, 1}};    /**< array of parameterset, access via gParamSetIdx*/
 
 /* EXTERNAL FUNCTIONS *****************************************************************************/
 
 void SetParameters_SetNextParamSet(void)
 {
-
+    gParamSetIdx++;
+  
+    if (3 >= gParamSetIdx)
```

Comment:
3 <=
yoda style sucks big time! It solves as many problems as it creates

magic number? even if not a constant could make sense